test/calendar_test.js

```diff
@@ -181,11 +181,11 @@ describe("Calendar", () => {
            'type': 'JRTT'
           },
           {'date': utcDate(2017, 1, 5),
-           'hours': 4,
+           'hours': 8,
            'type': 'JRTT'
           },
           {'date': utcDate(2017, 1, 6),
-           'hours': 4,
+           'hours': 8,
```

Comment:
actually I think this was a mistake, this test was really to check a full week of holiday. And actually this case wasn't working properly anymore with the new holiday range I chose, because of the week-end issue above